modules/EvseManager/EvseManager.cpp

```diff
@@ -167,52 +167,43 @@ void EvseManager::ready() {
-            // transfer_modes.push_back(types::iso15118_charger::EnergyTransferMode::DC_core);
             transfer_modes.push_back(types::iso15118_charger::EnergyTransferMode::DC_extended);
 
-            powersupply_capabilities = r_powersupply_DC[0]->call_getCapabilities();
-
-            // subscribe to run time updates e.g. due to de-rating
-            r_powersupply_DC[0]->subscribe_update_capabilities(
-                [this](auto caps) { update_powersupply_capabilities(caps); });
-
-            updateLocalMaxWattLimit(powersupply_capabilities.max_export_power_W);
-
-            setup_physical_values.dc_current_regulation_tolerance =
-                powersupply_capabilities.current_regulation_tolerance_A;
-            setup_physical_values.dc_peak_current_ripple = powersupply_capabilities.peak_current_ripple_A;
-
+            // Init power supply capabilities with safe defaults
+            types::power_supply_DC::Capabilities psu_caps;
+            psu_caps.bidirectional = false;
+            psu_caps.peak_current_ripple_A = 0.5;
+            psu_caps.current_regulation_tolerance_A = 0.5;
+            psu_caps.max_export_current_A = 0;
+            psu_caps.max_export_power_W = 0;
+            psu_caps.max_export_voltage_V = 60;
+            psu_caps.min_export_voltage_V = 0;
+            psu_caps.min_export_current_A = 0;
+
+            // Set present measurements on HLC to sane defaults
             types::iso15118_charger::DC_EVSEPresentVoltage_Current present_values;
             present_values.EVSEPresentVoltage = 0;
             present_values.EVSEPresentCurrent = 0;
             r_hlc[0]->call_update_dc_present_values(present_values);
 
-            setup_physical_values.dc_energy_to_be_delivered = 10000;
-
-            types::iso15118_charger::DC_EVSEMaximumLimits evseMaxLimits;
-            evseMaxLimits.EVSEMaximumCurrentLimit = powersupply_capabilities.max_export_current_A;
-            evseMaxLimits.EVSEMaximumPowerLimit = powersupply_capabilities.max_export_power_W;
-            evseMaxLimits.EVSEMaximumVoltageLimit = powersupply_capabilities.max_export_voltage_V;
-            setup_physical_values.dc_maximum_limits = evseMaxLimits;
-            charger->inform_new_evse_max_hlc_limits(evseMaxLimits);
+            // apply sane defaults capabilities settings once on boot
+            update_powersupply_capabilities(psu_caps);
 
-            types::iso15118_charger::DC_EVSEMinimumLimits evseMinLimits;
-            evseMinLimits.EVSEMinimumCurrentLimit = powersupply_capabilities.min_export_current_A;
-            evseMinLimits.EVSEMinimumVoltageLimit = powersupply_capabilities.min_export_voltage_V;
-            setup_physical_values.dc_minimum_limits = evseMinLimits;
+            // subscribe to run time updates for real initial values (and changes e.g. due to de-rating)
+            r_powersupply_DC[0]->subscribe_capabilities([this](auto caps) { update_powersupply_capabilities(caps); });
```

Comment:
Why doesn't the subscribe happen earlier?  Doing this during `ready` we might miss the capabilities if the dc power meter is quick.

Reply:
Furthermore, the lambda should have `(const auto& caps)`.